packages/@expo/cli/src/start/server/BundlerDevServer.ts

```diff
@@ -308,6 +309,11 @@ export abstract class BundlerDevServer {
     return this.instance;
   }
 
+  private requireAppLaunchMode(): AppLaunchMode {
```

Comment:
Xit: getAppLaunchMode

require is reserved for actions which involve the require keyword